apps/backend/src/libs/modules/tokenizer/base-tokenizer.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Tokenizer } from "./libs/types/types.js";
+
+class BaseTokenizer implements Tokenizer {
```

Comment:
```suggestion
class BaseToken implements Token {
```
will be enought. do not forget about file name